app/scripts/migrations/050.js

```diff
@@ -0,0 +1,32 @@
+import { cloneDeep } from 'lodash'
+
+const version = 50
+
+const LEGACY_LOCAL_STORAGE_KEYS = [
+  'METASWAP_GAS_PRICE_ESTIMATES_LAST_RETRIEVED',
+  'METASWAP_GAS_PRICE_ESTIMATES',
+  'cachedFetch',
+  'BASIC_PRICE_ESTIMATES_LAST_RETRIEVED',
+  'BASIC_PRICE_ESTIMATES',
+  'BASIC_GAS_AND_TIME_API_ESTIMATES_LAST_RETRIEVED',
```

Comment:
`BASIC_GAS_AND_TIME_API_ESTIMATES` is missing:

```suggestion
  'BASIC_GAS_AND_TIME_API_ESTIMATES',
  'BASIC_GAS_AND_TIME_API_ESTIMATES_LAST_RETRIEVED',
```